packages/example-codehub/src/controllers/health-controller.api.ts

```diff
@@ -3,18 +3,28 @@
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
 
-export const def = {
+import {OpenApiSpec} from '@loopback/core';
+
+export const def: OpenApiSpec = {
+  swagger: '2.0',
+  info: {
+    title: 'unused',
+    version: 'unused',
+  },
```

Comment:
>has to include mandatory fields from the root OpenAPI spec object

Can we define our own type -- ApiDefinition (which looks pretty much the same and makes the OAS root props optional and maps spec.paths accordingly)?